app_test.go

```
@@ -879,46 +880,113 @@
+	t.Run("timeout with long running request", func(t *testing.T) {
+		t.Parallel()
+		app := New()
+
+		app.Get("/", func(c Ctx) error {
+			time.Sleep(2 * time.Second)
+			return c.SendString("OK")
+		})
+
+		ln := fasthttputil.NewInmemoryListener()
+		go func() {
+			_ = app.Listener(ln)
```

golangci-lint (1.62.2)

Error return value of `app.Listener` is not checked

(errcheck)